src/root.js

```diff
@@ -98,10 +98,10 @@ Root.prototype.load = function load(filename, options, callback) {
         /* istanbul ignore if */
         if (!callback)
             return;
-        var cb = callback;
-        callback = null;
         if (sync)
             throw err;
+        var cb = callback;
```

Comment:
The style is internally consistent, so LGTM. But, it would be an improvement to use `let` / `const` to protect us from weird scope issues.